azure/app/template.json

```diff
@@ -183,6 +205,59 @@
         }
       }
     },
+
+    {
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2017-05-10",
+      "name": "[variables('vspAppServicePlanDeploymentName')]",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('deploymentUrlBase'), 'app-service-plan.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "appServicePlanName": {
+            "value": "[variables('vspAppServicePlanName')]"
+          }
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Web/sites",
+      "apiVersion": "2016-08-01",
+      "name": "[variables('vspAppServiceName')]",
+      "kind": "app,linux,container",
+      "location": "[resourceGroup().location]",
+      "dependsOn": ["[variables('vspAppServicePlanDeploymentName')]"],
+      "properties": {
+        "httpsOnly": true,
+        "serverFarmId": "[variables('vspAppServicePlanId')]",
+        "siteConfig": {
+          "alwaysOn": "[parameters('appServiceAlwaysOn')]",
```

Comment:
If this is set to `false`, we might have pretty slow responses on the first request after a deploy. Just a heads up.

Reply:
Happy to change this to `true` if you think it'd be better

Reply:
No, I think it's fine. Just thought it was worth pointing out.